Source/WebInspectorUI/UserInterface/Models/FontStyles.js

```diff
@@ -138,7 +138,10 @@ WI.FontStyles = class FontStyles
 
         const createIfMissing = true;
         let cssProperty = this._effectiveWritablePropertyForName(targetPropertyName, createIfMissing);
+
         cssProperty.rawValue = targetPropertyValue;
+        if (this._nodeStyles._propertyNameToEffectivePropertyMap[cssProperty.name]?.important)
```

Comment:
```diff
- this._nodeStyles._propertyNameToEffectivePropertyMap[cssProperty.name]?.important
+ this._nodeStyles._propertyNameToEffectivePropertyMap.get(cssProperty.name)?.important
```

Reply:
Changed to: `this._nodeStyles._propertyNameToEffectivePropertyMap.get(cssProperty.name)?.important`